.gitattributes

```diff
@@ -1,3 +1,5 @@
+* text=auto
```

Comment:
Just to clean up this file a bit, should the plain text entries below (ones without a diff clause) be removed?

Reply:
Yeah we probably should removed the redundancy.